domains/kan.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "ishigami"
+    },
+    "record": {
+        "A": "193.123.102.231"
```

```suggestion
        "A": ["193.123.102.231"]
```
A records should be in an array.